specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/education.json

```diff
@@ -299,7 +299,7 @@
             "$ref": "./examples/LabIncludeBudget.json"
           }
         },
-        "operationId": "Get_Lab",
+        "operationId": "Labs_Get",
```

Comment:
`Labs_List` and `Labs_Get` has the same description: `"Get the details for a specific lab associated with the provided billing account name, billing profile name, and invoice section name."` which may confuse customers.